src/common/util/game_def.hpp

```diff
@@ -3,6 +3,12 @@
 
 #include <SFML/Graphics.hpp>
 
+enum State {
```

Comment:
everything about this change looks fine, but please please don't call this thing State. It's the equivalent of calling it Thing. 

Also none should be probably be not_started right? If we frame it that way we can call it GameStatus or something

Reply:
Changed it to GameStatus